examples/admin-remove-user.ts

```diff
@@ -2,7 +2,7 @@ import { SpvWalletClient } from '../dist/typescript-npm-package.cjs.js';
 import { exampleAdminKey, examplePaymail } from './example-keys.js';
 import { errMessage } from './utils.js';
 
-const server = 'http://localhost:3003/v1';
+const server = 'http://localhost:3003/';
```

Comment:
it shouldn't end with `/`

Reply:
Actually the `httpclient` implementation allows both options (with and without trailing slash)

```golang
// httpclient.ts
private prepareUrl(path: string): string {
    path = path.startsWith('/') ? path.substring(1) : path;
    return this.baseUrl + path;
  }
```

But I agree that we should be consistent among all examples 👌 so I'm good 😄

Reply:
should be resolved